File: engine/model/src/main/scala/tools/packet/spawn/RemoveMonsterInvisibility.scala
package tools.packet.spawn

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoveMonsterInvisibility(private var _monsterObjectId: Int) extends PacketInput {
  def monsterObjectId: Int = _monsterObjectId

  override def opcode(): SendOpcode = SendOpcode.SPAWN_MONSTER_CONTROL
}

File: engine/engine-model/src/main/java/tools/packet/spawn/RemoveMonsterInvisibility.java
package tools.packet.spawn;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoveMonsterInvisibility(Integer monsterObjectId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.SPAWN_MONSTER_CONTROL;
   }
}